Config/default_aws_resource_mappings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "AWSResourceMappings": {},
+    "AccountId": "",
+    "Region": "us-west-2",
+    "Version": "1.0.0"
+}
```

Comment:
I don't think the region should be defaulted in this file.

Also the file is missing an ending newline.

Reply:
Fixed!